example/lib/line_chart/samples/line_chart_sample2.dart

```diff
@@ -80,6 +80,7 @@ class _LineChartSample2State extends State<LineChartSample2> {
         bottomTitles: SideTitles(
           showTitles: true,
           reservedSize: 22,
+          rotateAngle: 45,
```

Comment:
And please do not change our pages 1, 2, 3 samples, they are our showcases and we do not try new features in them, you can try it in bar_chart_sample5.